.github/COMPONENT-GUIDE.md

```diff
@@ -336,6 +267,78 @@ ion-button {
 ```
 
 
+### Activated
+
+The activated state should be enabled for elements with actions on "press". It usually changes the opacity or background of an element.
+
+> [!WARNING]
+>`:active` should not be used here as it is not received on mobile Safari unless the element has a `touchstart` listener (which we don't necessarily want to have to add to every element). From [Safari Web Content Guide](https://developer.apple.com/library/archive/documentation/AppleApplications/Reference/SafariWebContent/AdjustingtheTextSize/AdjustingtheTextSize.html):
+>
+>> On iOS, mouse events are sent so quickly that the down or active state is never received. Therefore, the `:active` pseudo state is triggered only when there is a touch event set on the HTML element
+
+> Make sure the component has the correct [component structure](#component-structure) before continuing.
```

Comment:
```suggestion
Make sure the component has the correct [component structure](#component-structure) before continuing.
```

I don't think you wanted to use a quote block here.

Reply:
I didn't add this section with this PR, I just moved the section which is why it looks like this was a new addition. I can update all of these though.

Reply:
Updated in https://github.com/ionic-team/ionic-framework/pull/29111/commits/c1797b87b65aeebf1d30eecd244e272b8629e8aa